### ISSUE TITLE
Wakatime working offline error, online dashboard seems fine

### ISSUE THREAD

**Issue by ivancuric** [wakatime/vscode-wakatime#223]
For the past few days I don't have the "Working offline..." error in the statusbar and I don't see the time tracking. The online dashboard seems fine. I've attached the debug log:

[wakatime.log](https://github.com/wakatime/vscode-wakatime/files/7015195/wakatime.log)

**Comment by codeybench** [comment]
Seems like the "working offline" error appears only on vscode 1.59.1. I have another machine with installed vscode 1.59.0, waka works perfectly fine there

I've handled this error with deletion of proxy line in the config file and with setting up the debug mode to true. But the problem is that I have statistics on my status bar now but I still have the same error message on hover the bar. Moreover, the time logger incorrectly track another project, which is based on another machine with vscode v. 1.59.0. Unfortunately, logger doesn't know that I am switching between projects every day. There was no problem with it until I've upgraded my vscode or waka
